<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>747</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>布匹瑕疵检测系统</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Icons/MainForm.png</normaloff>Icons/MainForm.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="CameraA">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
      <height>320</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>摄像头A</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="CameraB">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>640</width>
      <height>320</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>摄像头B</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="CameraC">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>320</y>
      <width>640</width>
      <height>320</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>摄像头C</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableView" name="tV_info">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>320</y>
      <width>640</width>
      <height>320</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_start">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>650</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>开始检测</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_end">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>650</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>停止检测</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_picture">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>650</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>文件检测</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_statistics">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>650</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>数据统计</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_settings">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>650</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>参数设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_CamAsettings">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>650</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>设置摄像头A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_CamBsettings">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>650</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>设置摄像头B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_CamCsettings">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>650</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>设置摄像头C</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="User"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionsetA">
   <property name="text">
    <string>setA</string>
   </property>
  </action>
  <action name="User">
   <property name="text">
    <string>用户手册</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
